packages/components/src/DataTable/DataTable.css

```diff
@@ -0,0 +1,106 @@
+.dataTable {
+  display: flex;
+  flex-direction: column;
+
+  width: 100%;
+  border: 1px solid rgb(225, 225, 225);
+  color: rgb(66, 78, 86);
+}
+.dataTable .tableContainer {
+  /* When fixed height */
+  max-height: 50vh;
+  overflow: scroll;
+}
+
+.dataTable .tableContainer table {
+  width: 100%;
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
+.dataTable .tableContainer thead {
+  border-bottom: 2px solid rgb(225, 225, 225);
+  background-color: white;
+
+  /* When fixed height
+  position: sticky;
+  top: -1px;
+  left: 0;
+  right: 0;
+  */
+}
+
+.dataTable .tableContainer tbody {
+  position: relative;
+}
+
+.dataTable .tableContainer th {
+  text-align: left;
+  cursor: pointer;
+}
+.dataTable .tableContainer th:hover,
+.dataTable .tableContainer th:first-child:hover {
+  background-color: rgb(239, 239, 239);
+}
+
+.dataTable .tableContainer th > div {
+  padding-left: 17px;
+}
+
+.dataTable .tableContainer tr {
+  position: relative;
+  height: 52px;
+  border-top: 1px solid rgb(225, 225, 225);
+  border-bottom: 1px solid rgb(225, 225, 225);
+}
+
+.dataTable .tableContainer td {
+  padding-left: 17px;
+}
+
+.dataTable .tableContainer td:first-child {
+  font-weight: bold;
+}
+
+/* Detect with screen size AND javascript in case it overflows even on desktop?*/
+@media only screen and (max-width: 768px) {
```

Comment:
We have preset [breakpoints](https://atlantis.getjobber.com/breakpoints) you can use